setup.py

```diff
@@ -29,7 +29,7 @@
     author_email="opensource@prodigygame.com",
     license="MIT",
     packages=["python_graphql_client"],
-    install_requires=["aiohttp==3.7.3", "requests==2.22.0", "websockets==8.1"],
+    install_requires=["aiohttp>=3.0.0", "requests>=1.0.0", "websockets>=5.0"],
```

Comment:
This looks good but I'm a bit concerned that these libraries release a new major version with lots of breaking changes since that's what major versions are for. So, I was just wondering what would be the pip equivalent of the caret(^) range of npm. I guess we can do something like `aiohttp==3.*.*` to limit the major version?

Reply:
For sure, we can definitely do that! I think the `~=` is what would work here.

```suggestion
    install_requires=["aiohttp~=3.0", "requests~=2.0", "websockets>=5.0,<9"],
```

I initially went with a looser restriction on versions since I wasn't sure if it was better to be preventative or reactive to issues caused by major version changes of dependencies. If we restrict to a major version here, say `requests v2`, then we will need to actively monitor and respond to major version changes of that project. Really the tradeoff is whether we want to avoid any dependency issues and manually test a version bump of a dependency when it comes out or if we want to react to issues as people identify them with allowed versions of dependencies. I think there's merit to both approaches, and the issues I see with manually checking for dependency version bumps is less of a factor if we have something like `dependabot` or another similar service giving us alerts when a new major version comes out.

Reply:
Yeah, it sounds like you have great insight into the trade-offs. Leveraging tools like `dependabot` or `renovate` for automating dependency management sounds like a good idea. 👍